dateutil/test/test_tz.py

```diff
@@ -1079,6 +1082,28 @@ def testGettzCacheTzLocal(self):
         assert local1 is not local2
 
 
+@pytest.mark.gettz
+def test_gettz_same_result_for_none_and_empty_string():
+    local_from_none = tz.gettz()
+    local_from_empty_string = tz.gettz("")
+    assert local_from_none is not None
+    assert local_from_empty_string is not None
+    assert local_from_none == local_from_empty_string
+
+
+@pytest.mark.gettz
+@pytest.mark.parametrize("gettz_arg", [None, ""]) # could add colon here
+@given(dt=st.datetimes(max_value=datetime.now()))
+def test_gettz_returns_local(gettz_arg, dt):
+    dt_act = dt.replace(tzinfo=tz.gettz())
+    dt_exp = dt.replace(tzinfo=tz.tzlocal())
```

Comment:
I mostly used your suggestion, but used only `tzlocal()` since `astimezone()` sets a `timezone` instance that returns `None` for `dst()`